packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -153,6 +183,19 @@ const Updates = decorate([
       toggleState,
       update: () => xoaUpdater.update(),
       upgrade: () => xoaUpdater.upgrade(),
+      getReleaseChannels: async () => {
+        const xoaReleaseChannels = await xoaUpdater.getReleaseChannels()
+        return { xoaReleaseChannels }
+      },
```

Comment:
Inline this in `initialize`.

Reply:
This is an affect that update the state. Certainly you want me to inline it for a reason, can I know it ?

Reply:
- `initialize` is also an effect
- `getReleaseChannels` is only use once and it does not make sense to call it again by itself

Reply:
I see, thank you for your answer :+1: